jenkins-integration/beaker/install/shared/install_deps.rb

```diff
@@ -0,0 +1,9 @@
+step "Configure epel" do
+  tmp_module_dir = master.tmpdir('configure_epel')
+  on(master, puppet('module', 'install', 'stahnma-epel', '--codedir', tmp_module_dir))
+  on(master, puppet('apply', '-e', "'include epel'"))
```

Comment:
@rlinehan Thanks again for finding this.  I added b031427.  I didn't run that back through a full Gatling job setup but I did use the beaker test in this repo to install puppet and then ran the install_deps.rb test.  The test passed for me, installing EPEL as expected and nuking the temp module directory when done.